### PR TITLE
=pro do not fork stream tests since #16544

### PR DESCRIPTION
**Pull request by 2m**
This fixes the issue where system properties where not propagated to forked jvm.

### CODE REVIEW

**Comment by 2m** [comment]
Failure was another incarnation of #17301

**Comment by patriknw** [comment]
Refs #17301

File: project/AkkaBuild.scala [hunk 1]
```diff
@@ -602,9 +602,7 @@ object AkkaBuild extends Build {
     settings = defaultSettings ++ formatSettings ++ scaladocSettings ++ experimentalSettings ++ javadocSettings ++ Seq(
       version := streamAndHttpVersion,
       libraryDependencies ++= Dependencies.streamTest,
-      previousArtifact := None,
-      // FIXME remove this when persistence tests are moved out
```
Comment (patriknw) [inline project/AkkaBuild.scala]: ah, that was an old one

**Comment by patriknw** [comment]
LGTM

**Comment by ktoso** [comment]
LGTM

**Comment by drewhk** [comment]
LGTM

**Pull request merged**